packages/orama/src/methods/search.ts

```diff
@@ -223,6 +224,7 @@ export async function search<AggValue = Result[]>(
       uniqueDocsArray = docsWithIdAndScore.map(([id, score]) => [id, score])
     } else {
       uniqueDocsArray = await orama.sorter.sortBy(orama.data.sorting, uniqueDocsArray, params.sortBy)
+        .then(results => results.map(([id, score]) => [getInternalDocumentId(orama.internalDocumentIDStore, id), score]))
```

Comment:
I don't like this map here. Why not put it inside sortBy?

Reply:
It will require me to change the `.sortBy` to return just `InternalID`, I didn't want to change because of https://github.com/oramasearch/orama/pull/441#discussion_r1261368165

But if you are okay with introducing this breaking change, I can change and it will not require anymore the `.map`.

Reply:
keep like that.